var/spack/repos/builtin/packages/hip/package.py

```diff
@@ -214,7 +219,16 @@ class Hip(CMakePackage):
     patch(
         "0011-Improve-compilation-without-git-repo-and-remove-compiler-rt-linkage-for-host"
         ".5.0.2.patch",
-        when="@5.0.2:",
+        when="@5.0.2:5.1.3",
+    )
+
+    # Improve compilation without git repo and remove compiler rt linkage
```

Comment:
What does "improve" mean in this context? Looking at the patch, it seems to ne this allows compilation without `git`, am I correct?

Reply:
The new patch address a build failure with 5.2.0 release sources in addition to the git related changes that were carried from the initial 3.5.0 release which we get merged to 5.3.0 soon.